sourcegraph/about
# Case study: Nutanix

- Closes #5209 for `/about`.
- **Note**: I will add this to `/re-platform` repo in the more ideal way following the refactored patterns started for Case Studies.
- Adds a typography partial for font colors not covered by Bootstrap utility classes
- Adds a 'Back' button to all case studies' jumbotron
- Makes `CaseStudyCard` more flexible to be used on index page & as preview at bottom of Case Study page

<img width="932" alt="image" src="https://user-images.githubusercontent.com/59381432/160906545-6a33e698-4a52-444c-a707-2d851d375e41.png">
<img width="1233" alt="image" src="https://user-images.githubusercontent.com/59381432/160906489-a17439b4-72f8-41af-b04b-27535f78678c.png">

### Test
- Navigate to `/case-studies/nutanix-fixed-log4j-with-sourcegraph`
- Ensure spec matches on web & mobile
- Ensure links & meta work

I'm all ears for feedback since this introduces a lot of new formatting to this section of About and I want to make sure I'm falling in line with our desired patterns. :)
